packages/http-server/src/server.ts

```diff
@@ -1,15 +1,26 @@
 import { configMergerFactory } from '@stoplight/prism-core';
 import { createInstance, IHttpMethod, ProblemJsonError, TPrismHttpInstance } from '@stoplight/prism-http';
 import * as fastify from 'fastify';
-import { IncomingMessage, Server, ServerResponse } from 'http';
+// @ts-ignore
+import * as fastifyAcceptsSerializer from 'fastify-accepts-serializer';
+import { IncomingMessage, ServerResponse } from 'http';
 import { getHttpConfigFromRequest } from './getHttpConfigFromRequest';
 import { IPrismHttpServer, IPrismHttpServerOpts } from './types';
 
 export const createServer = <LoaderInput>(
   loaderInput: LoaderInput,
   opts: IPrismHttpServerOpts<LoaderInput>,
 ): IPrismHttpServer<LoaderInput> => {
-  const server = fastify<Server, IncomingMessage, ServerResponse>();
+  const server = fastify().register(fastifyAcceptsSerializer, {
+    serializers: [
+      {
+        regex: /json$/,
+        serializer: JSON.stringify,
```

Comment:
Should this use `safeStringify`, from `@stoplight/json`?

Reply:
According to the documentation, the only thing `safeStringify` does more than the regular function is to handle circular references — which is a thing you can't express in an OAS document (as in a text file in general), so I skipped that. Does that make sense?

Reply:
Yup, all good then!